BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -98,21 +107,32 @@ void BSP_CAN_Init(void) {
     NVIC_InitStructure.NVIC_IRQChannelSubPriority = 0x0;
     NVIC_InitStructure.NVIC_IRQChannelCmd = ENABLE;
     NVIC_Init(&NVIC_InitStructure);	
+
+
+    //return CANinitSuccess;
+    
 }
 
 /**
  * @brief   Transmits the data onto the CAN bus with the specified id
- * @param   id : Message of ID. Also indicates the priority of message. The lower the value, the higher the priority.
- * @param   data : data to be transmitted. The max is 8 bytes.
- * @param   length : num of bytes of data to be transmitted. This must be <= 8 bytes or else the rest of the message is dropped.
- * @return  0 if module was unable to transmit the data onto the CAN bus. Any other value indicates data was transmitted.
+* @param   id : Message of ID. Also indicates the priority of message. The lower the value, the higher the priority.
+* @param   Voltage : the voltage value at that moment in a float
```

Comment:
This comment needs to be updated, since we are sending the voltage as an integer with units of millivolts